scripts/jenkins/ed.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Kops requires user input through an editor. This shell script simply returns
+# the file input '$1' to STDOUT which enables us to script editing nodes through
```

Comment:
Looking at how this is used it's actually that we produce a fake editor for every "edit" with kops that always just writes the same file (`$1`) to the "to-edit" file (`$2`), which is probably some tmp file kops controls.